actiontext/CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+*  The `fill_in_rich_text_area` system test helper locates a Trix editor and fills it in with the given HTML:
+
+   ```ruby
+   # <trix-editor id="message_content" ...></trix-editor>
+   fill_in_rich_text_area "message_content", with: "Hello world!"
+
+   # <trix-editor placeholder="Your message here" ...></trix-editor>
+   fill_in_rich_text_area "Your message here", with: "Hello world!"
+
+   # <trix-editor aria-label="Message content" ...></trix-editor>
+   fill_in_rich_text_area "Message content", with: "Hello world!"
+
+   # <input id="trix_input_1" name="message[content]" type="hidden">
+   # <trix-editor input="trix_input_1"></trix-editor>
+   fill_in_rich_text_area "message[content]", with: "Hello world!"
+   ```
```

Comment:
Could reinforce that you're filling in HTML, not plain text: `with: "Hello <em>world!</em>"`